src/mappings/erc721/erc721-mapping.ts

```diff
@@ -69,28 +73,8 @@ export async function handleERC721(event: TransferLog): Promise<void> {
     await nft.save();
   }
 
-  const transferId = getTransferId(
-    chainId,
-    event.transactionHash,
-    event.logIndex.toString(),
-    0
-  );
-
-  const transfer = Transfer.create({
-    id: transferId,
-    tokenId: event.args.tokenId.toString(),
-    amount: BigInt(1),
-    networkId: chainId,
-    block: BigInt(event.blockNumber),
-    timestamp: event.block.timestamp,
-    transaction_hash: event.transactionHash,
-    nftId: nft.id,
-    from: event.args.from.toLowerCase(),
-    to: event.args.to.toLowerCase(),
-  });
-
   await Promise.all([
-    transfer.save(),
+    handle721Transfer(chainId, event, nft.id),
```

Comment:
Its a bit inconsistent that `handle721Transfer` saves the entitiy but `handle1155Transfer` returns the entity to be saved